README.md

```diff
@@ -117,8 +117,16 @@ alias FUCK='fuck'
 Or in `config.fish`:
 
 ```fish
-function fuck
-    eval (thefuck $history[1])
+function __thefuck_repl --description 'Replace operators into fish-compatible'
+    set -l tmp (echo $argv | sed 's/ && / ; and /g')
+    echo $tmp | sed 's/ || / ; or /g'
+end
+
+function fuck --description 'Correct your previous console command'
+    set -l eval_script /tmp/.thefuck.fish
```

Comment:
Hi,

I'd like to add you to my professional network on LinkedIn.

- R

Confirm that you know R: https://www.linkedin.com/e/v2?e=-vzcnar-i99jtls3-2p&amp;a=preRegInvite&amp;tracking=eml-guest-invite-cta&amp;ek=invite_guest&amp;sharedKey=s6w-pACt&amp;invitationID=6000889217104773120


You received an invitation to connect. LinkedIn will use your email address to make suggestions to our members in features like People You May Know. Unsubscribe here: https://www.linkedin.com/e/v2?e=-vzcnar-i99jtls3-2p&amp;t=lun&amp;midToken=AQFmrFVhGEd36Q&amp;ek=invite_guest&amp;loid=AQErCKscOMsdpAAAAU0dxCmGDljlAbEvVjwoZVCVd8sfXxIrRpV4LVpMT4upVN0b8JoV0xD_8RgmMF1SrVZYCwuJGydNEJpUbM4-t8bkthh1UE1ivDFkeaWLUMizUiZbIQZSTyckiSi0kV6O0a2-J-s9aPiCzeUJhLVdqNsvenv_YktBLHXAyeJQFfoqaMVBcoh5&amp;eid=-vzcnar-i99jtls3-2p Learn why we included this at the following link: https://www.linkedin.com/e/v2?e=-vzcnar-i99jtls3-2p&a=customerServiceUrl&ek=invite_guest&articleId=4788
&copy; 2015, LinkedIn Corporation. 2029 Stierlin Ct. Mountain View, CA 94043, USA